components/modules/Kconfig

```diff
@@ -110,6 +110,12 @@ config LUA_MODULE_LEDC
   help
       Includes the LEDC module.
 
+config LUA_MODULE_MQTT
+  bool "MQTT module"
+  default "n"
+  help
+      Includes the MQTT module.
```

Comment:
I noticed that when selecting MQTT in menuconfig that it was the only component that was not in alphabetical order so I reordered it.